grc/sidekiq_sidekiq_rx.block.yml

```diff
@@ -68,6 +84,8 @@ outputs:
 - label: Samples
   domain: stream
   dtype: complex
+  multiplicity: ${ (2 if (port_id2 != '100') else 1) }
```

Comment:
Should this be `handle2` instead of `port_id2`?

Reply:
Maybe this is just a weird naming thing, but I think this should be consistent with the naming in template.  Either `handle1` and `handle2` used or `port_id1` and `port_id2` used.  Thoughts?  I may be way off here and thinking about this all wrong, just let me know.

Reply:
Good catch, I changed everything to handle1 handle2 because it is a handle not a port name.

I wonder why this actually worked?